app/views/timecards/show.html.erb

```diff
@@ -35,7 +35,7 @@
   %>
 <% end %>
 
-<table class="datatable table table-striped table-bordered" >
+<table class="generic-datatable table table-striped table-bordered" >
```

Comment:
I thought this `generic` was what caused the heading issue in another table?

Reply:
@kgf I am not aware of a heading issues within another table, do you know which one was having issues? I'm pretty sure I resolved all of the heading issues.